src/browser/api_protocol/api_handlers/authorization.js

```diff
@@ -123,14 +124,14 @@ function onRequestAuthorization(id, data) {
 
         rvmMessageBus.registerLicenseInfo({
             data: {
-                licenseKey: externalConnObj.licenseKey,
-                client: externalConnObj.client,
+                licenseKey: authObj.authReqPayload && authObj.authReqPayload.licenseKey,
+                client: authObj.authReqPayload && authObj.authReqPayload.client,
```

Comment:
Same in these case as in the comment above...?

Reply:
Added check for authObj